### PR TITLE
Update Google Cast iOS sender SDK

### PR DESCRIPTION
**Pull request by 123mpozzi**
## Description

Update Google Cast iOS sender SDK to match the version used in the player

## Checklist
- [x] 🗒 `CHANGELOG` entry

### CODE REVIEW

File: CHANGELOG.md
Comment (rolandkakonyi) [inline CHANGELOG.md:8]:
```suggestion
- Update example app dependency: Google Cast iOS sender SDK to `4.8.1`
```

Comment (123mpozzi) [reply CHANGELOG.md:8]: Thank you! I was struggling to phrase this 😅

**Review by rolandkakonyi — APPROVED** [review]
Please do not merge before the iOS Player SDK update PR is merged!

**Pull request merged**
